api/schema/resolvers/PaymentResolver.js

```diff
@@ -126,6 +140,86 @@ module.exports = {
                 }
             })
             return models.Payment.findByPk(id)
+        },
+        convertUSDtoSATS: async (root, { amount }, { models }) => {
+            try {
+                // Convert the USD amount to SATS
+                const sats = await bitcoinConversion.fiatToSatoshis(amount, 'USD')
+                
+                return sats
+        
+            } catch (error) {
+                throw new Error('Failed to convert USD to SATS: ', error);
+            }
+        },
+        sendPayment: async (root, { contributors }, { models }) => {
```

Comment:
consider refactoring the control for this function to depend more on the order of the `contributors` array instead of trying to create 3 separate arrays for IDs, amounts, and wallets